bitnami/jupyterhub/Chart.yaml

```diff
@@ -26,4 +26,4 @@ name: jupyterhub
 sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/jupyterhub
   - https://github.com/jupyterhub/jupyterhub
-version: 3.0.13
+version: 3.0.14
```

Comment:
You don't need to bump the chart version, tests are not packed as part of the chart.